pypeit/core/flexure.py

```diff
@@ -303,147 +305,226 @@ def spec_flex_shift(obj_skyspec, arx_skyspec, arx_lines, mxshft=20):
                 corr_cen=corr.size/2, smooth=smooth_sig_pix, success=success)
 
 
-def spec_flexure_obj(specobjs, slitord, bpm, method, sky_file, mxshft=None):
-    """Correct wavelengths for flexure, object by object
+def spec_flexure_slit(slits, slitord, slitmask, bpm, sky_file, method="boxcar", specobjs=None, slitspecs=None,
+                      waveimg=None, mxshft=None, slitShift=None):
```

Comment:
Please add some docs here explaining the boxcar vs slitcen functionality, as the information below is not enough to understand what the difference is.